main.go

```diff
@@ -136,6 +140,12 @@ func cmd() *cobra.Command {
 		},
 		Run: func(_ *cobra.Command, args []string) {
 			err := upgrade.UpgradeProvider(context, repoOrg, repoName)
+			if err != nil {
```

Comment:
If not running in GitHub CI, skip the issue creation?

Reply:
We can reuse the `pulumi-infer-version` flag for this